Codeinwp/neve
# Dynamic tags for HFG components

### Summary
Adds magic tags to HFG components - HTML & Button.


### Will affect the visual aspect of the product

NO

### Test instructions


- Check the inserter on the HTML & Button components works fine;
- It should work fine with and without WooCommerce;
- It should work fine with this PR of Neve Pro Addon: https://github.com/Codeinwp/neve-pro-addon/pull/453 - you will have to update the `NEVE_VERSION` constant to `2.5.4` for this to work properly;
- You can quickly test all magic tags by pasting this into the HTML component:
```
current_single_title - {current_single_title}
current_single_excerpt - {current_single_excerpt}
archive_description - {archive_description}
archive_title - {archive_title}
site_title - {site_title}
site_tagline - {site_tagline}
author_bio - {author_bio}
author_name - {author_name}
current_single_url - {current_single_url}
home_url - {home_url}
archive_url - {archive_url}
author_url - {author_url}
current_year - {current_year}
product_price - {product_price}
product_title - {product_title}
cart_link - {cart_link}
checkout_link - {checkout_link}
```


Closes Codeinwp/neve-pro-addon#428.
